libs/vkd3d/vkd3d_private.h

```diff
@@ -1964,7 +1965,9 @@ struct d3d12_graphics_pipeline_state
     VkPipelineRasterizationDepthClipStateCreateInfoEXT rs_depth_clip_info;
     VkPipelineRasterizationStateStreamCreateInfoEXT rs_stream_info;
 
-    uint32_t dynamic_state_flags; /* vkd3d_dynamic_state_flag */
+    /* vkd3d_dynamic_state_flag */
+    uint32_t explicit_dynamic_states;
```

Comment:
Is explicit_dynamic_states really necessary? Seems like we could just look at pipeline_dynamic_states?

Reply:
Having a separate `explicit_dynamic_states` seems more robust to me, `pipeline_dynamic_states` is only written at the time the pipeline gets compiled and may or may not depend on things like GPL support.